app/controllers/get_involved_controller.rb

```diff
@@ -37,48 +37,56 @@ def retrieve_given_document_type(document_type)
 
   # Aims to find a count of consultations closed in the last n months.
   def retrieve_date_filtered_closed_consultations(months)
-    closed_count = 0
+    cutoff_date = Time.zone.now.prev_month(months)
 
-    cl_docs = retrieve_special({document_type: 'closed_consultation', per_page: 500, fields: ['details']})['results']
-    cl_docs.sort_by! { |k| -k['details']['closing_date'] }.reverse!
+    query = {
+      filter_display_type: "Closed consultation",
```

Comment:
Same as above, `filter_content_store_document_type: "closed_consultation"` would be better here.